data/mape/mape.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+np.random.seed(0)
+M=100
+y_hat = np.random.normal(loc=10,scale=3,size=(M,))
+y = np.random.normal(loc=9, scale=2, size=(M,))
+
+test_mape = abs((y - y_hat)*100/ y).mean()
```

Comment:
Can you check this equation? Afaik, this should be `np.mean(abs((y - y_hat)/y)* 100)`

Reply:
The .mean() function is from the math library of python and it works.
Should I change it to numpy?

Reply:
Keeping it as is for now.